plugins/in_winevtlog/in_winevtlog.c

```diff
@@ -238,6 +238,11 @@ static struct flb_config_map config_map[] = {
       0, FLB_TRUE, offsetof(struct winevtlog_config, use_ansi),
       "Use ANSI encoding on eventlog messages"
     },
+    {
+      FLB_CONFIG_MAP_BOOL, "use_ansi", "false",
+      0, FLB_TRUE, offsetof(struct winevtlog_config, ignore_missing_channels),
+      "Use ANSI encoding on eventlog messages"
```

Comment:
Could you use the actual description and parameter here like as `ignore_missing_channels` or similar?

Reply:
Description is now updated